firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionModel.swift

```diff
@@ -178,15 +178,31 @@ class TrackingProtectionModel {
 
         let confirmAction = UIAlertAction(title: clearCookiesAlertButton,
                                           style: .destructive) { [weak self] _ in
-            self?.clearCookiesAndSiteData(cookiesClearable: CookiesClearable(), siteDataClearable: SiteDataClearable())
+            self?.clearCookiesAndSiteData()
             self?.selectedTab?.webView?.reload()
+
+            guard let windowUUID = self?.selectedTab?.windowUUID else { return }
+            store.dispatch(
+                TrackingProtectionMiddlewareAction(
+                    windowUUID: windowUUID,
+                    actionType: TrackingProtectionMiddlewareActionType.dismissTrackingProtection
+                )
```

Comment:
I see we are dispatching an action to dismiss the sheet (presumably so the user can see the toast that appears behind it), but when testing, the sheet does not dismiss. I don't see where [navigateTo .close is being handled](https://github.com/mozilla-mobile/firefox-ios/blob/865545339db4fca6c4271fb8978645f534e8a530/firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionState.swift#L256).

Reply:
thank you for reviewing it. The dismiss shouldn't work because the connection to the navigation is in another https://github.com/mozilla-mobile/firefox-ios/pull/23666. (unless that one gets merged first and I could rebase this one)

Reply:
@MattLichtenstein are there any more concerns here that need addressing?

Reply:
That was all - R+